.github/workflows/clang-tidy.yml

```diff
@@ -23,7 +23,7 @@ jobs:
         config_file: src/.clang-tidy
         build_dir: build
         apt_packages: libopenmpi-dev,libhdf5-mpi-dev,python3-dev,python3-numpy,python3-matplotlib
-        cmake_command: cmake . -B build -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DQUOKKA_PYTHON=ON -DQUOKKA_OPENPMD=ON -DopenPMD_USE_ADIOS2=OFF
+        cmake_command: cmake . -B build -DCMAKE_EXPORT_COMPILE_COMMANDS=ON -DQUOKKA_PYTHON=ON -DQUOKKA_OPENPMD=ON -DopenPMD_USE_ADIOS2=OFF -DAMReX_SPACEDIM=3
         split_workflow: true
```

Comment:
@BenWibking clang-tidy will pass now. The main issue was that clang-tidy built cmake in 1D, and would review popiii.cpp. But the PopIII test is only built in 3D so that'd why clang-tidy could not find some of the header files. 

However, we perhaps do not want clang-tidy to always run in 3D for every PR. So, what's the long term solution?

Reply:
Nice debugging!

The goal of all of the testing is to make sure there aren't any bugs in production simulations, so I think running clang-tidy with `AMReX_SPACEDIM=3` is fine. The only real downside is that it won't catch missing `AMREX_D_DECL` macros for indices that might cause 1D test to fail.